src/libse/Forms/FixCommonErrors/FixDanishLetterI.cs

```diff
@@ -21,77 +21,47 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
                 var oldText = text;
 
                 // Make sure text contains lower: 'i'.
-                if (RegexUtils.LittleIRegex.IsMatch(text))
+                if (callbacks.AllowFix(p, Language.FixDanishLetterI))
                 {
-                    foreach (var regex in RegexUtils.DanishLetterI.DanishCompiledRegexList)
+                    if (RegexUtils.LittleIRegex.IsMatch(text))
```

Comment:
The issue was that combining `callbacks.AllowFix(p, Language.FixDanishLetterI)` and `RegexUtils.LittleIRegex.IsMatch(text)` prevent the execution to enter the statement!

I added few tests also